koans/StringManipulationKoans.php

```diff
@@ -52,35 +56,40 @@ public function testInterpolatingAssociativeArrayElementsMustBeInCurlyBrackets()
     }
 
     /**
-     * @testdox Heredoc interpolates like a double-quoted string.
+     * @test
+     * @testdox Heredoc interpolates like a double-quoted string
      */
-    public function testHeredocInterpolatesLikeADoubleQuotedString()
+    public function heredocInterpolatesLikeADoubleQuotedString()
     {
         $value = "one";
         $string = <<<EOT
-The value is $value
-EOT;
+        The value is $value
+        EOT;
```

Comment:
This is not a string

Reply:
No? How strange because testing in the tests it takes me as a string.